pkg/cloud-provider/securitygroup/securitygroup.go

```diff
@@ -104,10 +104,10 @@ Antrea internal NetworkPolicy To SecurityGroup Mapping strategy
 // CloudResourceType specifies the type of cloud resource.
 type CloudResourceType string
 
-const (
-	NepheControllerPrefix             = "nephe-"
-	NepheControllerAddressGroupPrefix = NepheControllerPrefix + "ag-"
-	NepheControllerAppliedToPrefix    = NepheControllerPrefix + "at-"
+var (
+	ControllerPrefix             *string
```

Comment:
Dont use pointer.. store the variable in string

Reply:
We should use it as a pointer that's how we are updating the name.